src/firebaseConfig.js

```diff
@@ -1,13 +1,15 @@
 // Import the functions you need from the SDKs you need
 import { initializeApp } from "firebase/app";
 import { getAuth } from "firebase/auth";
+import { getDatabase, ref } from "firebase/database";
 // TODO: Add SDKs for Firebase products that you want to use
 // https://firebase.google.com/docs/web/setup#available-libraries
 
 // Your web app's Firebase configuration
 const firebaseConfig = {
   apiKey: "AIzaSyCZom64UGQVE98VSCQRDmnH6XSl1vgGgeY",
```

Comment:
Thanks I'm aware of the need to hide api keys, I just don't know how to do it in a team setting so that others can still use it. 

If you know how to set up hiding the api key but still making it available for team members to use please let me know! I think that whole firebaseConfig variable should be hidden as the URL is also sensitive info.

Reply:
Perhaps we could share sensitive info in the Teams Google drive? It's possible to create a new file for that purpose.

https://drive.google.com/drive/folders/1DFhQ3sZ1MyA57dT4L2GFcmSq-xxxaaGp

Reply:
Could do that but I think figuring out environmental variables with github might be more productive. We can add it as an issue and honestly I think its fine for them to just be public for now. there's no sensitive data and the account was created for this purpose only. Worst case I can refresh the api key. 

We can also discuss on monday's call if you prefer.